discovery-provider/tests/test_model_validator.py

```diff
@@ -0,0 +1,79 @@
+import logging
+
+from jsonschema import ValidationError
+from src.model_validator import ModelValidator
+
+logger = logging.getLogger('model_validator')
+
+###### Testing field validation with variation of instances ######
+def test_one_field_schema_pass():
+    track = {"title": "ok"}
+    try:
+        ModelValidator.validate(to_validate=track, field="title", model="Track")
+    except ValidationError as e:
+        assert False, f"test_model_validator [test_one_field_schema_pass] failed: {e}"
```

Comment:
what's up with the assert False? why do we do that?

Reply:
the test was intended to validate a valid instance with the schema. if a validation error occurs, fail the test

Reply:
ah, i see it just passes on the asset because you catch it here. ok makes sense! never seen this pattern before, but seems reasonable